timemachine/fe/rbfe.py

```diff
@@ -792,7 +791,7 @@ def run_edges_parallel(
             edge,
             mols,
             ff,
-            protein,
+            protein_path,
```

Comment:
This should be the protein object, the path won't be accessible on a remote server.

Reply:
Ah, thanks. I was seeing an error from the remote jobs that indicated they were trying to read in the PDB file, so I thought the driver script would need to store it using the remote file client and pass the path into the job. But looks like the error might've been due to another bug. Will revert this.